### ISSUE TITLE
Connecting profiler_gui after capture has been started & stopped programmatically does not result in data being downloaded

### ISSUE THREAD

**Issue by mmallet-youilabs** [yse/easy_profiler#153]
Consider the following case:

1. Run an application
2. This application programmatically starts recording (EASY_PROFILER_ENABLE)
3. After some time, the application programmatically stops recording (EASY_PROFILER_DISABLE)
4. profiler_gui is connected to the still-running application

At this point, the application has recorded profiling data but that data isn't transmitted to the gui app.

To receive the data, the user has to use the Capture button in the GUI. However this causes _new_ data to be captured, in addition to the previously-captured data. There is no way to get the previously-captured data only.

**Comment by cas4ey** [comment]
Yeah, currently profiler_gui will only receive the data if the application is profiling to the moment of profiler_gui connection.  
#todo
